docker/.env

```
@@ -407,6 +407,37 @@ DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=1
 # Used for ThreadPoolExecutor
 THREAD_POOL_MAX_WORKERS=128
 
+# -----------------------------------------------------------------------------
+# Knowledge compilation
+# -----------------------------------------------------------------------------
+# Wiki LLM pool and phase timeouts. Values are read when the Python service
+# starts; restart the service after changing them.
+WIKI_MAP_LLM_POOL_SIZE=20
+WIKI_MAP_MAX_PENDING=25
+WIKI_REFINE_WORKERS=20
+WIKI_MAP_WORKERS=20
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The WIKI_MAP_WORKERS key should go before the WIKI_REFINE_WORKERS key

(UnorderedKey)